dev/src/ComponentIntegration/Command/ComponentIntegration.php

```diff
@@ -272,12 +272,38 @@ private function testComponent(array $component)
         $commands = [
             "cd {$component['tmpDir']}",
             "composer update --no-suggest",
-            "echo \"\\nRUNNING UNIT TESTS\\n\"",
-            "vendor/bin/phpunit",
-            "echo \"\\nRUNNING SNIPPET TESTS\\n\"",
-            "vendor/bin/phpunit -c phpunit-snippets.xml.dist"
         ];
 
+        if (file_exists($component['tmpDir'] . '/tests/Unit')) {
```

Comment:
Could we abstract this into a helper function?

Reply:
done!